cpu/samd5x/include/candev_samd5x.h

```diff
@@ -169,7 +169,7 @@ typedef struct {
     /** CAN device configuration descriptor */
     const can_conf_t *conf;
     /** CAN message RAM accessible to the CAN controller */
-    msg_ram_conf_t msg_ram_conf;
+    struct can_msg_ram msg_ram;
```

Comment:
```suggestion
    can_msg_ram_t msg_ram;
```

might as well use it

Reply:
all consider the _t should not exist.